app/controllers/api/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::SessionsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def show 
+    current_user ? head(:ok) : head(:unauthorized)
+  end
+
+  def create 
+    user = User.where(email: params[:email]).first
```

Comment:
If you're looking for just one record, is better to use `user =  User.find_by(email: params[:email])`
check this doc for more info https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find